### DISCUSSION TITLE
Funder - Collect applicant and project information via application form

### DISCUSSION THREAD

**Discussion by fjjulien** [pair-cg/general-contributions#83] [User Stories]
# 1. Motivating Story
_What's the story here? Please describe why you have a particular need.
I'm a program manager at the Department of Canadian Heritage. I run a program called the [Canada Arts Presentation Fund](https://www.canada.ca/en/canadian-heritage/services/funding/arts-presentation-fund/programming-arts-festivals-series-presenters.html). The first thing we are doing in each fiscal year cycle is to receive application information populated by applicants into an [application form](https://www.canada.ca/en/canadian-heritage/services/funding/arts-presentation-fund/programming-arts-festivals-series-presenters/forms-programming-professional-arts-festivals-performing-arts-series-presenters.html).

# 2. Need:
_Now the why is out of the way, what is it you actually need?_
We have three main needs:

1. Assess the eligibility of the applicant
2. Assess the eligibility of the proposed project
3. Assess the relative merit of the proposed project vis-à-vis the program objectives.

# 3. Action:
_What is an action that can be taken towards fulfilling that need?_

The first action that we took when we launched the program was to design an application form that would cover all our information needs. As I think back of the form creation process, how the form evolved over time and what didn't change, it strikes me that we have many information needs beyond the three main needs that I stated above. 

1. Part A and B of the application forms what we call the "tombstone". We call it that way because it holds identification information that hardly ever changes (and yet applicants have to fill it anew year after year). It essentially holds all legal information necessary to unambiguously identify who the applicant is (legal name and usual name, incorporation number), what their legal status is, whether they are a charity, their address and who is the authorized signatory. In other words, it identifies the contracting party with whom the department may eventually sign a contribution agreement.
2. Part A and B also holds statistical and self-identification information about the applicant. Self-identification information describes the type of activities generally presented by the organization and for which kinds of audiences. Most importantly, we want to know if they have a mandate to serve target groups for the Department: ethnocultural, Aboriginal, official-language minority or youth. This self identification is however unsubstantiated and needs to be verified. It's not uncommon to see organizations claiming to have a mandate to serve all four target groups, but then, as we look at their website, we find little evidence of activities illustrating this claim. Even we we explicitly suggest they shouldn't check all boxes, they do it again the next year. I wish it was possible to ask for documentation from which we could assert once and for all whether they do have a mandate to serve these target groups.
3. Part D asks for information about the programming, including whether it is a series or a festival, and the total anticipated attendance. 
4. Then, we need a complete schedule of activities. For each anticipated performance or outreach activity, the applicant must provide: name of the artist/group/company, where they are from, the number of performances, what discipline/genre, and whether they fit any of our target groups (called "programming categories"). Again, we would need a means for easily verify the programming categories claims. Some are blatantly inaccurate. Others fall into grey zones. Sometimes the same artists can be categorized differently by two applicants. Sometimes, there's intersectionality: an artist may be Black and be part of an official-language minority. Unfortunately our form cannot deal with intersectionality: an artist can only be attributed a single category. 
5. At the end of the funding cycle, applicants must populate the same schedule of activities, but with additional information: total attendance and fees paid to the artists. We're also asking them to aggregate this information based on the province or country of residence of the artists. I guess this information could be inferred from the schedule of activities, but we can't do that ourselves easily.
6. We also require a budget, of course

# 4. Desired Outcome:
_What is the desired outcome?_

Applicants find the application and reporting processes onerous: they keep populating some of the same information over and over; and there's no way that they can reuse any of this information (i.e. pre-populate their schedule of activities in their final report). If any of this information could be populated once and then reused, we wouldn't need to verify it each time. Everyone would save time. Ideally, identification related to target groups should be populated only once. And when it's about artists, the artists themselves should be given the opportunity to self-identify. I believe that's the direction the federal government wants to take the Disaggregated Data Action Plan announced in the federal budget. I look forward to learning more. 

# 5. Perspective:
_From what perspective is this user story coming from?_
The story is written primarily from the perspective of the funder, but attempts to also elicit data pains and gains from the point of view of the applicant. In this particular program, the applicants are presenting organizations.

# 6. Stakeholder Group:
_Which stakeholder groups are impacted by this User Story?_
The data inefficiencies in this story are so blatant that they could be deem to impact the entire performing arts value chain. Applicants and the funder waste lots of time populating and processing information and they nonetheless end up with poor quality data that doesn't fully meet their operational needs. And, of course, no one can reuse any of this data for other purposes such as research. Sigh!

# 6. Additional info:
_Here's more information._

**Comment by drosu** [comment]
Hi @fjjulien, 

Thank you for this funder-perspective story. I tried to convert it into a use case and I have a few questions I am hoping you can answer:

Q1. The information listed under "Action" focuses on the deficiencies of the current software the Department is using, e.g., 
- the web form used to collect information does not support
     -> intersectionality, i.e., an artist can only be listed as a member of a single category
 - aggregation of financial information provided by applicants by geographic location (province / country) is not supported
 - pre-populating fields of the final report form (e.g., pre-populate the info related to the schedule of activities) is not supported
 information that the Department of Canadian Heritage collects from applicants for the Canada Arts Presentation Fund

so I am wondering if it would be accurate to say that the actions the Department of Canadian Heritage takes in order to fulfill its stated needs are in fact:
- collect information from the applicant (via a web-form)
- verify the accuracy / truthfulness of the information provided, e.g.,
         -> verify the "programming categories" claims. 
         -> verify the accuracy of the self-identification information
- determine applicant/ proposal eligibility and proposal merit (based on the information provided and verified)?

Q2. Is the following an accurate and complete description of the information collected by the Department?
 - legal information (about the applicant)
 - self-identification information
 - programming information

2.1 legal information about the applicant:
   - legal name
   - usual name
   - incorporation number
   - legal status (e.g., charity)
   - address 
   - authorized signatory

2.2 self-identification information:
    -  type of activities presented by the organization 
    -  demographic

2.3 programming
   -  type of programming, e.g., series, festival
   - total anticipated attendance (as number of people)
   - budget
   - applicants must populate the same schedule of activities
   - total actual attendance 
   - total fees paid to the artists
   - total fees paid to the artists by province of residence of the artists
   - total fees paid to the artists by country of residence of the artists
   
   - programming schedule (e.g., start date and end date ? time? duration? for each programming component, i.e., performance or outreach activity)
   
   2.3.1 performance information 
            - name of the artist/group/company performing
            - discipline/genre of the performance
  
   2.3.2  outreach activity  information
   **Q: What is the difference between a performance and an outreach activity? (the information collected for both seems to be the same)**

   2.3.3 artist/group/company engaged / invited to perform
            - geographic origin / location of the artist/group/company, e.g., Canadian province, Country
            - number of performances the artist/group/company will be delivering
             - "compatibility with purpose" (_whether they fit any of the target groups identified by the Department, i.e., ethnocultural, Aboriginal, official-language minority or youth_ )

Thank you!

**Reply by fjjulien** [reply]
Hi Daniela,

Q1. The main actions are indeed to collect the application information via the form and then assessing the eligibility and merit. Program officers do check applicant's self-identification claims, but they don't have enough time to verify "programming category" claims. I was however stressing this information to highlight two deficiencies: 
a) The form isn't designed to handle intersectionality. 
b) It shouldn't be up to each applicant to associate a "programming category" to each artist on their application form. These "programming categories" should be defined by the artists themselves and this information should be stored in some place where it can be retrieved for each application where a given artist is listed.
The same is also true of self-identification by the applicant organization. This information shouldn't need to be provided for each application. It should be stored some open repository, where the claims can be substantiated with sources (and validated or invalidated by the community).

Q2. The list you have is quite accurate. I will nonetheless provide a few additional notes:

- "Incorporation number": While this particular program relies on the incorporation number as an identifier, many other programs are starting to collect the Business Number, because it is a better candidate for a unique identifier across Canada.
- "legal status (e.g., charity)": The form does not specifically ask for the legal status or legal form. However, this information needs to be asserted by the program officer because certain legal forms such as for-profit corporation are not eligible. As to charitable status, from a representational point of view it is not a legal form per se and it needs to be modeled with a distinct property from legal form. For more information, see this extension of the [LDFI Conceptual model for organizations](https://docs.google.com/spreadsheets/d/12a94EggXA1FdvlV2mBe-A_b_lHzxZNnGze5MO6akrRY/edit?usp=sharing). This spreadsheet has bilingual labels and descriptions for several properties which I have found to be frequently used by funders.
- The form also asks if the applicant is a member of a presenter support organization or network, if so, which one(s)? Membership in associations is a useful information to assert if an organization is recognized by their peers, is following certain industry standards, and/or is contributing to their community. Yet, it is a difficult information to find. Associations are notoriously inefficient at tracking their own membership information and they cannot even be trusted to publish accurate membership information on their website (very few associations have member directories that are really up-to-date). Membership information should also be qualified with start time and end time metadata. Few associations keep records of start times and end times (even CAPACOA hasn't been diligent enough in this regard).
- "Outreach activities": The [CAPF Glossary](https://www.canada.ca/en/canadian-heritage/services/funding/arts-presentation-fund/programming-arts-festivals-series-presenters/application-guidelines-programming-professional-arts-festivals-performing-arts-series-presenters.html#a24) defines outreach activities as: "Audience development/outreach is a process that may involve two stages. The first consists of identifying, informing, researching, reaching and engaging a new audience through specific promotional activities, market research, audience profiling and establishing contacts in particular sectors of the community. Once the audience has been identified and engaged, the second stage consists of building knowledge and appreciation of specific artistic disciplines or forms, presenting new artists or disciplines, and then developing the demand for such presentations. This is achieved through public discussion/lectures, pre- or post-show panel discussions, roundtable presentations, workshops open to the public, demonstrations, residencies, seminars, public rehearsals and other forms of contact with the community by professional artists programmed by the presenter in the community. All audience development/outreach activities must be directed to the general public." As you can see, the term is too broadly defined to be represented as a single concept. Moreover, the performing arts sector has been moving away from this term because it is perceived to be centred on the privilege holder reaching out to those who may not share the privilege of having access to the arts. A more common term nowadays is "community engagement". However, it too lacks a clear definition. For a research project conducted in 2017, we chose not to avoid either term and we rather defined a [list of specific activities](https://docs.google.com/spreadsheets/d/1hMAOLoWb-KbQIWAC_Q4Rw2XbVoqMLFd6EnHidFnw4JI/edit?usp=sharing) that were deemed to fall under either the "outreach" or the "community engagement" category. This list was defined in consultation with practitioners and academics. While I don't deem it perfect or definitive, it was an important contribution to defining non-performance public activities in the performing arts.

I hope this is useful.

Frédéric
